.github/workflows/ros-workflow.yml

```diff
@@ -272,6 +245,7 @@ jobs:
     - name: Commit raise version
       id: raise
       run: |
+        git config --global --add safe.directory $(pwd)
```

Comment:
what is that for ?

Reply:
Git just released a new security rule. 
https://github.blog/2022-04-12-git-security-vulnerability-announced/


![Screenshot from 2022-04-22 15-32-39](https://user-images.githubusercontent.com/84720623/164735744-74e7a472-59f5-48a0-8f16-eb8f495e11b8.png)